src/server/routes/eventbrite.routes.js

```diff
@@ -0,0 +1,411 @@
+const routes = require('express').Router();
+const apiKeys = require('../config/config.js');
+const rp = require('request-promise');
+
+// GET /api/yelp
+// Description: get one or more businesses from eventbrite with query params
+// q: q is a keyword for search
+// locationAddress: address | zipcode | city or state
+// locationLongitude & locationlatitude
+//
+// Examples: 
+// eg. https://www.eventbriteapi.com/v3/events/search/?token=ALPXUSBHYYOPXWXOSA54
+// eg. https://www.eventbriteapi.com/v3/events/search/?token=ALPXUSBHYYOPXWXOSA54&location.address=Manhattan&location.within=5mi
+
+routes.get('/api/eventbrite', (req, res) => {
+
+  const q = req.query.q || undefined
+  const locationAddress = req.query.location || undefined
+  const locationLongitude = locationAddress ? undefined : req.query.longitude || undefined
+  const locationlatitude = locationAddress ? undefined : req.query.latitude || undefined
```

Comment:
Make sure variable names follow consistent conventions.  the L in Latitude should be capitalized.